package.json

```diff
@@ -26,6 +26,7 @@
     "es6-promise": "^2",
     "eslint": "^1.9.0",
     "eslint-plugin-no-unsafe-innerhtml": "^1.0.14",
+    "eslint-plugin-no-wildcard-postmessage": "0.1.2",
```

Comment:
My main questions were:
* did you see real violations in our codebase ? The ignores you added seem to be quite legit (but I only looked quickly).
* I wondered about false positives (an unrelated method called postMessage). But the risk is low that its second argument could accept `'*'` in a legit way anyway.